README.md

```diff
@@ -375,36 +360,37 @@ To use Regula with Conftest:
 
 ### Directory structure
 
- -  `bin/`: the main Regula script that calls `terraform` & `opa`.
+ -  `bin/`: the main Regula script that, depending on the IaC platform, will call `opa` and `terraform` or convert CloudFormation YAML to JSON.
  -  `lib/`: the OPA library code to evaluate rules and mangle input.
- -  `rules/`: a collection of rules.  We may split this up further as the number
-    of rules increases.
+ -  `rules/`: a collection of rules.
  -  `examples/`: a collection of example rules that you can use as inspiration
      for your own rules.
  -  `scripts/`: scripts for development; currently only a script to generate
     test input.
  -  `tests/`:
       *  `tests/lib`: internal tests for the library.
       *  `tests/rules/`: tests for the various rules.
-      *  `tests/rules/inputs`: terraform files that can be used to generate Rego
+      *  `tests/rules/tf/<provider>/<service>/inputs`: tests for Terraform rules
+      *  `tests/rules/cfn/<service>/inputs`: tests for CloudFormation rules
+      *  `tests/rules/inputs`: CloudFormation and Terraform files that can be used to generate Rego
          files.
       *  `tests/examples/`: tests for the example rules.
       *  `tests/examples/inputs`: input files for the example rules.
 
 ### Adding a test
 
 If you would like to add a rule, we recommend starting with a test.
-Put your terraform code in a file in `tests/rules/<provider>/inputs`; for example
-[tests/rules/aws/inputs/kms\_rotate\_infra.tf](tests/rules/aws/inputs/kms_rotate_infra.tf).
+Put your code in a file in `tests/rules/tf/aws/kms/inputs`; for example
```

Comment:
Does it make sense to tell people to put their files in `tests/rules/tf/aws/kms/inputs` here or can we generalize, maybe like "in the `inputs` folder within the appropriate `tests/rules` directory"?